client/my-sites/domains/domain-management/edit/style.scss

```diff
@@ -49,10 +49,34 @@
 	}
 }
 
+.domain-only-cta__button-row {
+	margin: 17px 0;
+
+	button {
+		padding: 8px 34px;
+		font-size: $font-body-small;
+		font-weight: normal;
+
+		&.is-primary {
```

Comment:
This should be the standard Calypso button - no need to add the shadow and change font size and etc. Recently the default button styles changed and I guess the screenshot you're looking at was just using the old styles.